dashboard/app/controllers/admin_hoc_controller.rb

```diff
@@ -9,53 +10,54 @@ class AdminHocController < ApplicationController
   include LevelSourceHintsHelper
 
   def event_signups
+    # A hash from HOC year to HOC data.
+    event_signup_data = {2014 => nil, 2015 => nil, 2016 => nil}
     SeamlessDatabasePool.use_persistent_read_connection do
-      # Requested by Roxanne on 16 November 2015 to track HOC 2015 signups by day.
-      # Get the HOC 2014 and HOC 2015 signup counts by day, deduped by email and name.
-      # We restrict by dates to avoid long trails of (inappropriate?) signups.
-      data_2014 = DB[:forms].
-                  where('kind = ? AND created_at > ? AND created_at < ?', 'HocSignup2014', '2014-08-01', '2015-01-01').
-                  group(:name, :email).
-                  # TODO(asher): Is this clumsy notation really necessary? Is Sequel
-                  # really this stupid? Also below.
```

Comment:
Now, I'm more convinced that (a) there isn't a simpler way and (b) it isn't as clumsy as I previously considered it.